tests/page-objects/default/analytics/analytics.wdio.page.js

```diff
@@ -20,6 +20,10 @@ const targetNumberPercentCount = (targetElement) => targetElement.$('.body .targ
 
 const targetGoalValue = (targetElement) => targetElement.$('.body .count .goal p');
 
+const emptySelectionError = () => $('.content-pane .item-content.empty-selection.selection-error');
```

Comment:
this piece: `.content-pane .item-content.empty-selection` can be a constant and reuse the selector in `emptySelectionNoError`